.github/workflows/labeler.yml

```diff
@@ -28,7 +28,7 @@ jobs:
   triage:
     runs-on: ubuntu-22.04
     steps:
-    - uses: actions/labeler@v5
+    - uses: actions/labeler@v4
```

Comment:
Looks like GH workflow still uses it from main branch. Hence, the current PR build failed. We need to blindly merge this?